src/storage/v3/shard_rsm.cpp

```diff
@@ -695,17 +707,18 @@ msgs::ReadResponses ShardRsm::HandleRead(msgs::ScanVerticesRequest &&req) {
   for (auto it = vertex_iterable.begin(); it != vertex_iterable.end(); ++it) {
     const auto &vertex = *it;
 
-    if (start_ids == vertex.PrimaryKey(View(req.storage_view)).GetValue()) {
+    if (start_ids <= vertex.PrimaryKey(View(req.storage_view)).GetValue()) {
       did_reach_starting_point = true;
     }
 
     if (did_reach_starting_point) {
       std::optional<std::map<PropertyId, Value>> found_props;
 
+      const auto *const schema = shard_->GetSchema(shard_->PrimaryLabel());
```

Comment:
```suggestion
      const auto * schema = shard_->GetSchema(shard_->PrimaryLabel());
```

Reply:
But we want const pointer to const schema, why remove const and make the pointer mutable?

Reply:
Yes, that is true, however usually we don't write out the right-most const. If you think about our functions, I think I have never saw a `XXX *const` parameter in our functions.